src/main/java/teammates/ui/webapi/action/BasicFeedbackSubmissionAction.java

```diff
@@ -67,10 +65,6 @@ protected StudentAttributes getStudentOfCourseFromRequest(String courseId) {
      */
     protected void checkAccessControlForStudentFeedbackSubmission(
             StudentAttributes student, FeedbackSessionAttributes feedbackSession) {
-        if (student == null) {
-            throw new EntityNotFoundException(new EntityDoesNotExistException("Student does not exist"));
-        }
-
```

Comment:
This doesn't seem relevant to this PR since you're just adding tests.

Reply:
Yup. Will do this in the other PR.

Reply:
What you need to do is revert these changes to their original, so that if this PR is merged it wouldn't affect what you're doing in your other PR. 

If the new test cases added here however require that these changes take effect, let me know and I'll put this PR on hold until your other PR is resolved :)

Reply:
> What you need to do is revert these changes to their original, so that if this PR is merged it wouldn't affect what you're doing in your other PR.

Ok reverted.

> If the new test cases added here however require that these changes take effect, let me know and I'll put this PR on hold until your other PR is resolved :)

Yup the new test cases added requires those changes. The travis check for this PR would probably fail. Kindly put this PR on hold :)

Reply:
Can start working on this again when you want to now that the other PR is resolved